sdk/policy/arm-policy/src/models/parameters.ts

```diff
@@ -83,108 +88,209 @@ export const nextLink: OperationURLParameter = {
     serializedName: "nextLink",
     required: true,
     type: {
-      name: "String"
-    }
+      name: "String",
+    },
   },
-  skipEncoding: true
+  skipEncoding: true,
 };
 
-export const scope: OperationURLParameter = {
-  parameterPath: "scope",
+export const contentType: OperationParameter = {
+  parameterPath: ["options", "contentType"],
   mapper: {
-    serializedName: "scope",
+    defaultValue: "application/json",
+    isConstant: true,
+    serializedName: "Content-Type",
+    type: {
+      name: "String",
+    },
+  },
+};
+
+export const parameters: OperationParameter = {
+  parameterPath: "parameters",
+  mapper: PolicyDefinitionMapper,
+};
+
+export const subscriptionId: OperationURLParameter = {
+  parameterPath: "subscriptionId",
+  mapper: {
+    serializedName: "subscriptionId",
     required: true,
     type: {
-      name: "String"
-    }
+      name: "Uuid",
+    },
   },
-  skipEncoding: true
 };
 
-export const policyAssignmentName: OperationURLParameter = {
-  parameterPath: "policyAssignmentName",
+export const policyDefinitionName: OperationURLParameter = {
+  parameterPath: "policyDefinitionName",
   mapper: {
-    serializedName: "policyAssignmentName",
+    constraints: {
+      Pattern: new RegExp("^[^<>*%&:\\?.+/]*[^<>*%&:\\?.+/ ]+$"),
+    },
+    serializedName: "policyDefinitionName",
     required: true,
     type: {
-      name: "String"
-    }
-  }
+      name: "String",
+    },
+  },
 };
 
 export const apiVersion1: OperationQueryParameter = {
   parameterPath: "apiVersion",
   mapper: {
-    defaultValue: "2021-06-01",
+    defaultValue: "2023-04-01",
```

Comment:
@qiaozha this is a multiple api version SDK but since this is a preview I will not block their release and let me know if any concern!